articles/first-android-app/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/first-android-app/
+title: Creating Your First Android Application
+description: This article will be a step-by-step guide to creating your first android application. It will be a very simple application to give you a general knowledge of android apps
+author: diana-mutheu
+date: 2021-02-01T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/first-android-app/hero.jpg
+    alt: Android application image example
+---
+Application Softwares commonly known as apps refer to programs designed for end-users. They enable a user to pass a set of instructions to a computer using a built user interface instead of hard code. There are different types of applications; web, mobile and, desktop. In this article, we will look at mobile applications, specifically android applications.
+<!--more-->
+### Introduction
+Mobile applications are of two types due to the two mobile operating systems, namely Ios and Android.
+
+- Examples of phone brands that use android are Samsung, Google pixel, etc.
+- An example of a phone brands that use Ios is Apple.
+
+This article will be a step-by-step guide to creating your first android application. It will be a very simple application to give you a general knowledge of android apps. We can develop Android applications using `Java` or `Kotlin` programming languages, here we will use Java.
+
+### Table of Content
+- [Prerequisites](#prerequisites)
+
+- [New Terminologies](#new-terminologies)
+
+- [The steps to build your Application](#the-steps-to-build-your-application)
+
+- [Conclusion ](#conclusion)
+
+### Prerequisites
+1. Basic [Java programming language](https://www.w3schools.com/java/java_intro.asp).
+
+2. [Java Development Kit](https://www.oracle.com/java/) downloaded.
+
+3. [Android Studio](https://developer.android.com/studio) installed.
+
+> NOTE: Download according to the version of windows you are running, 64bit or 32bit.
+ According to Google, you need 2GB of RAM and above (4GB is recommended) to run Android Studio and Java since they are memory consuming and will slow down your PC.
+
+### New terminologies
+ - **Method** - A block of code that only runs when called and performs an action.
+
+ - **Activity** - This is one screen on an Android App's user interface.
+
+ - **Fragment** - It is a part of an Activity.
```

Comment:
This definition is vague. Consider using something like this *'A Fragment represents a reusable portion of your app's UI'*